Extensions/FileSystem/filesystemtools.ts

```diff
@@ -494,20 +567,48 @@ namespace gdjs {
       }
     };
 
+    export const loadStringFromFileAsyncTask = (
+      variable: gdjs.Variable,
+      loadPath: string,
+      resultVar: gdjs.Variable,
+      removeCRCharacters: boolean
+    ) =>
+      asyncFs
+        ? new gdjs.PromiseTask(
+            asyncFs
+              .readFile(loadPath, { encoding: 'utf8' })
+              .then((data) => {
+                if (data)
+                  variable.setString(
+                    removeCRCharacters ? data.replace(/\r/g, '') : data
+                  );
+                resultVar.setString('ok');
+              })
+              .catch((err) => {
+                resultVar.setString('error');
+                logger.error(
+                  "Unable to load the text file from path: '" +
+                    loadPath +
+                    "': ",
+                  err
+                );
+              })
+          )
+        : (resultVar.setString('error'), new gdjs.ResolveTask());
```

Comment:
Nitpicking: careful not to abuse this - while it seems convenient it's a pain to parse for the reader.